docs/manuals/studio/gui.md

```diff
@@ -72,6 +72,147 @@ set to the property is proposed to the user:
   <img src="../images/autocomplete_function.gif" width="75%"/>
 </p>
 
+## Page preview
+
+If you are used to working with Markdown in Visual Studio Code, you certainly came across the
+[Markdown preview](https://code.visualstudio.com/docs/languages/markdown#_markdown-preview)
+feature: pressing `Ctrl+Shift+V` in an editor that contains Markdown content creates a window
```

Comment:
Might be different depending on user os

Reply:
Right. That's how Microsoft documents it though.
But I get it. How do YOU do that on your Mac?